### PR TITLE
feat: clarify the original developers

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Include the name of the author of the derived package in the xml. This PR is related to https://github.com/hal-shu-sato/apm/issues/174 .

## How Has This Been Tested?

- The current version of apm is able to read this xml correctly.

## Checklist:

- [x] Updated the modification date in `mod.xml`.

### CODE REVIEW

File: README.en.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ This file contains data of plugins and scripts.
 - **/packages/package/overview**: The overview of the package (Up to 35 characters)
 - **/packages/package/description**: The description of the package
 - **/packages/package/developer**: The name of the developer of the package
+- /packages/package/originalDeveloper: The name of the developer of the package from which it was derived
```
Comment (hal-shu-sato) [inline README.en.md:42]:
```suggestion
- _/packages/package/originalDeveloper_: The name of the developer of the package from which it was derived
```

File: README.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ AviUtlと拡張編集Pluginのデータファイル
 - **/packages/package/overview**: パッケージの概要（35字以内）
 - **/packages/package/description**: パッケージの説明
 - **/packages/package/developer**: パッケージの開発者
+- /packages/package/originalDeveloper: 派生元パッケージの開発者
```
Comment (hal-shu-sato) [inline README.md:42]:
```suggestion
- _/packages/package/originalDeveloper_: 派生元パッケージの開発者
```

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
Thank you for the implementation.
I want to fix the README decorations.

**Comment by mitosagi** [comment]
Corrected.

**Review by hal-shu-sato — APPROVED** [review]
Thank you for fixing them.

**Pull request merged**